app/services/distribution_by_county_report_service.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class DistributionByCountyReportService
+  def get_breakdown(distributions)
+    breakdown_struct = Struct.new(:name, :region, :num_items, :amount)
```

Comment:
I'd have this as a constant above the method rather than defined in the method:

```
Breakdown = Struct.new(:name, :region, :num_items, :amount)
...
breakdowns['Unspecified'] = Breakdown.new("Unspecified", "ZZZ", 0, 0.00)
```